### ISSUE TITLE
InOb polyfill throws "sort is not a function"

### ISSUE THREAD

**Issue by dvoytenko** [ampproject/amphtml#11887]
See https://pantheon.corp.google.com/errors/8199952591715153852?service=default-cdn-1p-canary&time=P1D&filter&project=amp-error-reporting

```
Error: doubleclick Error executing onCreativeRender: c.sort is not a function. (In 'c.sort()', 'c.sort' is undefined)
at (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/src/intersection-observer-polyfill.js:201)
at (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/extensions/amp-a4a/0.1/refresh-manager.js:232)
at getIntersectionObserverWithThreshold_ (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/extensions/amp-a4a/0.1/refresh-manager.js:292)
at (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/extensions/amp-a4a/0.1/refresh-manager.js:217)
at a4a (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/extensions/amp-a4a/0.1/refresh-manager.js:168)
at (https://raw.githubusercontent.com/ampproject/amphtml/1508896929494/extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:871)
```

Please triage and lower priority (or close the bug) if not a critical issue.

**Comment by zhouyx** [comment]
https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/refresh-manager.js#L228 
Threshold should be an array instead of a string here. 

Talked to @glevitzky offline, he'll turn off experiment to unblock canary and fix the issue later.

**Comment by zhouyx** [comment]
Actually! Just tested the native InOb behavior. `thresholds` can be either a number or an array of numbers. I'll fix the issue on inOb polyfill side instead.
